### ISSUE TITLE
Zombie encounter has no guaranteed correct choice

### ISSUE THREAD

**Issue by cgreen-bitwise** [luceroweb/haunted-house-game#111]
## Devices
Device name:	[Custom Desktop PC](https://pcpartpicker.com/list/JxkYvf "PCPartPicker build")
Processor:	Intel Core i5-6600K 3.5 GHz Quad-Core Processor
System type:	Windows 10 Pro 64-bit operating system, x64-based processor
Resolution:	1920x1080 pixels, 16:9 ratio (~92 PPI density)

Device name:	[SM-G950U - Samsung Galaxy S21 5G (USA Unlocked)](https://www.gsmarena.com/samsung_galaxy_s21_5g-10626.php "GSMArena Page")
Processor:	Octa-core (1x2.84 GHz Kryo 680 & 3x2.42 GHz Kryo 680 & 4x1.80 GHz Kryo 680)
System type:	Android 11, One UI 3.1
Resolution:	1080 x 2400 pixels, 20:9 ratio (~421 ppi density)

Brave Browser Version 1.31.87 Chromium: 95.0.4638.54 (Official Build) (64-bit)

Firefox Browser Version 93.0 (20210927210923) (Official Build) (64-bit)

## Expected Behavior
The user assumes there should be a correct choice to defeat each monster encounter.

## Actual Behavior
The user can't push the zombie away, nor can they always win by choosing to attack or dodge it; both of these options have randomized success chances.

## Steps to Reproduce Issue
1. Begin game 
2. Continue searching the rooms until you get the zombie encounter 
3. Try pushing the zombie, note that it doesn't pass or fail the encounter. 
4. Try dodging the zombie. You may pass, or you may fail, and if you fail work back to this point and try again until you pass. 
5. Try attacking the zombie. You may pass, or you may fail, and if you fail work until you pass the encounter.

**Comment by luceroweb** [comment]
@ProjectErostribe is our Haunted House story editor. 
We added some variety in which action will result in passing an event in order to keep the player guessing. Myles, do you think we should make the action choices more consistent in order to meet the user's expectation?

cc @cgreen-bitwise @lreyes-qae

**Comment by ProjectErostribe** [comment]
@luceroweb Yes, I agree with this suggestion.  I will work on these changes.

**Comment by SmiJa** [comment]
@cgreen-bitwise We have decided to reject this ticket. The reason for rejection is that we built in a randomization to make the game a little bit harder for the player.

cc @ProjectErostribe @luceroweb @lreyes-qae
